### ISSUE TITLE
fsgan

### ISSUE THREAD

**Issue by ak9250** [shaoanlu/fewshot-face-translation-GAN#3]
thoughts on fsgan and there method for face swapping, results look impressive https://nirkin.com/fsgan/

**Comment by iperov** [comment]
Nothing impressive.
fsgan's faceswapping is much worse than existing methods.

Reenactment at 1:23 in the video https://youtu.be/BsITEVX6hkE?t=83
produce unstable background.
My new avatar model in DeepFaceLab produce more stable background https://www.youtube.com/watch?v=rRGOQpR_3hw

**Comment by ak9250** [comment]
@iperov good results, is your model few shot or does it have to be trained for each face pair?

**Comment by iperov** [comment]
@ak9250 not few shot. As regular deepfake.

**Comment by gstark0** [comment]
Waiting to see the code release, looks promising except that unstable background which is awful.

**Comment by lseiyjg** [comment]
Unstable background can be ignored, because it usually uses a face mask to swap the face. FSGAN is quite good as its demo shows.

**Comment by iperov** [comment]
@lseiyjg no, in reenactment approach background must be stable, because reenactment is not a face swap.

**Comment by szad670401** [comment]
@iperov  The background can be removed by face landmark.
No Few Shot methods like DeepFake may only help little in some instant environment.
Some application scenarios that have no time to wait for training.

**Comment by iperov** [comment]
@szad670401 bla bla bla. Have you anything to show?

**Comment by szad670401** [comment]
Are you stupid?
Have you anything to show? 
Is that the one modified from deepfake and used it to fool 10k stars?
